src/countdown/Countdown.tsx

```diff
@@ -0,0 +1,71 @@
+import "./_countdown.scss";
+
+import React, {useLayoutEffect, useRef} from "react";
+import classNames from "classnames";
+
+import useCountdownTimer from "../core/utils/hooks/useCountdownTimer";
+import List from "../list/List";
+import ListItem from "../list/item/ListItem";
+import {generateCountdownItems} from "../core/utils/time/timeUtils";
+
+export interface CountdownProps {
+  startDate: Date;
+  testid?: string;
+  countDownIntervalInSeconds?: number;
+  alwaysShowSeconds?: boolean;
+  onEnd?: () => void;
+  customClassName?: string;
+}
+
+export interface CountdownItem {
+  id: "days" | "hours" | "minutes" | "seconds";
+  count: string | number;
+}
```

Comment:
Let's create a `countdown.d.ts` and move these there

Reply:
I wonder if we need `title` property for `CountdownItem`. Users may want to use other languages to display. Then, you will be changing line 63 with `item.title` @yasincaliskan

Reply:
Actually, we can add an optional prop like;
```
title: {
  day: "days",
  hour: "hours",
  ...
}
```
If we need another language, we can override it. What do you think @edizcelik ?